sources/views/form.js

```diff
@@ -95,12 +94,18 @@ export default class FormView extends JetView {
         const form = this.$$("contactForm");
         const id = url[0].params.id;
 
-        if (id) {
-            const contact = contacts.getItem(id);
-            form.setValues(contact);
-        } else {
-            form.clear();
-        }
+        webix.promise.all([
```

Comment:
you do not need for this data every time in url change
you need to wait for the data 1 time
in config()
config() can return a promise

```js
config() {
        const _ = this.app.getService("locale")._;
        return webix.promise.all([
                contacts.waitData,
                statuses.waitData,
                countries.waitData
            ]).then(() => {
                return {
                view: "form",
               ...
```